roles/install-nfs/files/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nfs-client-provisioner
+  labels:
+    app: nfs-client-provisioner
+  # replace with namespace where provisioner is deployed
+  namespace: default
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: nfs-client-provisioner
+  template:
+    metadata:
+      labels:
+        app: nfs-client-provisioner
+    spec:
+      serviceAccountName: nfs-client-provisioner
+      containers:
+        - name: nfs-client-provisioner
+          image: quay.io/external_storage/nfs-client-provisioner:latest
+          volumeMounts:
+            - name: nfs-client-root
+              mountPath: /persistentvolumes
+          env:
+            - name: PROVISIONER_NAME
+              value: fuseim.pri/ifs
+            - name: NFS_SERVER
+              value: 10.10.10.60
```

Comment:
what is this ip address?

Reply:
@mkumatag 
The IP address `10.10.10.60` is a placeholder representing the private IP of the master node where the NFS server is set up. This value is a dummy and will be dynamically substituted with the actual IP.

You can find where it is referenced here: [deploy-nfs-client.sh.j2](https://github.com/valen-mascarenhas14/k8s-ansible/blob/0c4b1b126b0cdab648a7a311b452f02e312bb89c/roles/install-nfs/templates/deploy-nfs-client.sh.j2#L24):